trade-finance-manager-ui/templates/case/_macros/case-summary-ndb.njk

```diff
@@ -8,101 +8,105 @@
-        </div>
-      {% endif %}
-    </div>
-    <div class="case-summary-overview-container">
-      <div class="ukef-grid-column-one-sixth case-summary-overview-column">
-        <div class="case-summary-overview-item">
-          <div class="ukef-heading-xs">Product</div>
-          <div class="ukef-body-s" data-cy="ukef-product">{{ tfm.product | dashIfEmpty }}</div>
-        </div>
+        {% set supplierColumnClassName = 'case-summary-supplier' %}
 
-        <div class="case-summary-overview-item">
-          <div class="ukef-heading-xs">Bank</div>
-          <div class="ukef-body-s" data-cy="bank-name">{{ deal.bank.name | dashIfEmpty }}</div>
-        </div>
+        {% if deal.submissionDetails.buyerName %}
+          {% set supplierColumnClassName = 'case-summary-supplier chevron-right' %}
+        {% endif %}
 
-        <div class="case-summary-overview-item">
-          <div class="ukef-heading-xs">Date received</div>
-            <div class="ukef-body-s" data-cy="date-received">
-              {{ tfm.dateReceived | formatDateString("DD-MM-YYYY", "D MMMM YYYY") | dashIfEmpty}}
-            </div>
+        <div class="{{ supplierColumnClassName }}" data-cy="supplier-column">
+          <dt class="label">
+            {{ supplierType.render(deal.submissionDetails.supplierType) }}
+          </dt>
+          <dd class="ukef-heading-l" data-cy="exporter-name">{{ deal.exporter.companyName | dashIfEmpty }}</dd>
         </div>
-      </div>
 
-      <div class="case-summary-overview-column ukef-grid-column-one-sixth">
-        <div class="case-summary-overview-item">
-          <div class="ukef-heading-xs">Type</div>
-          <div class="ukef-body-s" data-cy="submission-type">Non-bank delegation</div>
-        </div>
+        {% if deal.submissionDetails.buyerName %}
+          <div class="case-summary-buyer" data-cy="buyer-name-column">
+            <dt class="label">Buyer</dt>
+            <dd class="ukef-heading-l" data-cy="buyer-name">{{ deal.submissionDetails.buyerName }}</dd>
+          </div>
+        {% endif %}
       </div>
+    </dl>
+    <dl class="ukef-list">
+      <div class="case-summary-overview-container">
+        <div class="ukef-grid-column-one-sixth case-summary-overview-column">
+          <div class="case-summary-overview-item">
+            <dt>Product</dt>
+            <dd data-cy="ukef-product">{{ tfm.product | dashIfEmpty }}</dd>
+          </div>
 
-      <div class="case-summary-overview-column govuk-grid-column-one-third">
-        <div class="case-summary-overview-item">
-          <div class="ukef-heading-xs">Export description</div>
-          <div class="ukef-body-s" data-cy="export-description">{{ deal.submissionDetails.supplyContractDescription | dashIfEmpty }}</div>
-        </div>
+          <div class="case-summary-overview-item">
+            <dt>Bank</dt>
+            <dd data-cy="bank-name">{{ deal.bank.name | dashIfEmpty }}</dd>
```

Comment:
This is an example of a `<dl>` where it'll be much harder to fix the nesting of divs.

Reply:
Thanks Oscar, I changed markup and added some css, looks like minimal html markup works fine.

P.S. Tim confirmed that it is fine to remove `Telephone` field.